drivers/overlay/ov_endpoint.go

```diff
@@ -97,6 +99,16 @@ func (d *driver) DeleteEndpoint(nid, eid string) error {
 	}
 
 	n.deleteEndpoint(eid)
+
+	link, err := netlink.LinkByName(ep.ifName)
```

Comment:
Can you please change `ep.IfName` to hold the host end and can you also rename `name1` and `name2` to more meaningful names so that we don't have any confusion?